ci-operator/config/openshift/assisted-service/openshift-assisted-service-master.yaml

```diff
@@ -250,9 +250,8 @@ tests:
     env:
       ASSISTED_CONFIG: |
         NUM_MASTERS=1
-        NUM_WORKERS=0
-        ISO_IMAGE_TYPE=minimal-iso
```

Comment:
the test uses minimal-iso?

Reply:
I don't think that's mandatory. I aligned with how it is configured on assisted-test-infra.
https://github.com/openshift/release/blob/38c4d30fcf6948afe627d8b9b6443831d287f9cf/ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml#L170-L178

In case it is needed, the configuration should be configured according to the scenario as @eliorerz already configured on test-infra code.

https://github.com/openshift/assisted-test-infra/blob/0dd21d8af776614cbd739070e9f554b6f54bc77f/discovery-infra/test_infra/utils/global_variables/global_variables.py#L16-L25

Reply:
well, you can't align with it without checking out the history:
https://github.com/openshift/release/commit/4ede205fe6211938999f0ee4311aee2744a723f2

so yes, open a task to do the change in test-infra